nextjs/src/app/[locale]/publishers/layout.tsx

```diff
@@ -1,14 +1,18 @@
 import Image from 'next/image';
 
 import UserProvider from '@/components/UserProvider';
+import { unstable_setRequestLocale } from 'next-intl/server';
 
 import NavDropdown from './NavDropdown';
 
 export default function NavigationLayout({
   children,
+  params,
 }: {
   children: React.ReactNode;
+  params: { locale: string };
 }) {
+  unstable_setRequestLocale(params.locale);
```

Comment:
is this meant to be temporary? i noticed this issue: https://github.com/amannn/next-intl/issues/663

Reply:
It's temporary, but it's meant to be removed in a future version of next-intl that hasn't been released yet.